int32 length
int32 mode
int32[] id
int32[] speed
int32[] position
int32[] type
int32[] acceleration

int32 Motion_Mode
char Motion_Num
